astropy/stats/lombscargle/implementations/utils.py

```diff
@@ -28,7 +28,6 @@ def bitceil(N):
     Roughly equivalent to int(2 ** np.ceil(np.log2(N)))
     """
     if hasattr(int, 'bit_length'):
```

Comment:
Couldn't you remove the condition and the `else` branch? Are there integers around without `bit_length`?